test/parallel/test-fs-rmSync-special-char.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const common = require('../common');
```

Comment:
```suggestion
require('../common');
```

Pretty sure linter would yell at this...

Reply:
OK. I will avoid linter's yelling. However, I saw some other tests have this exact line. `test-fs-rmdir-recursive-*` Should I modify those and run all the tests?

Reply:
Other tests assign it to a variable because they actually use `common.something`, this test doesn't make use of the properties from common, so the linter would complain about the unused variable.